.github/workflows/production.yml

```diff
@@ -16,29 +16,22 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: Write config
-      uses: schdck/create-env-json@v1
-      with:
-        file-name: 'src/config/config.json'
+    - name: NPM install
+      run: npm install
+    - name: Production build
+      run: npm run build
+      env:
         CLIENT_PASSPHRASE: ${{ secrets.LEIF_PRODUCTION_CLIENT_PASSPHRASE }}
         CLIENT_PRIVATE_KEY: ${{ secrets.LEIF_PRODUCTION_CLIENT_PRIVATE_KEY }}
         LEIF_APP_PUBLIC_KEY: ${{ secrets.LEIF_PRODUCTION_APP_PUBLIC_KEY }}
         LEIF_APP_AAT_SIGNATURE: ${{ secrets.LEIF_PRODUCTION_APP_AAT_SIGNATURE }}
-        DISPATCHERS: "https://node1.mainnet.pokt.network,https://node2.mainnet.pokt.network,https://node3.mainnet.pokt.network,https://node4.mainnet.pokt.network,https://node5.mainnet.pokt.network,https://node6.mainnet.pokt.network,https://node7.mainnet.pokt.network,https://node8.mainnet.pokt.network,https://node9.mainnet.pokt.network,https://node10.mainnet.pokt.network,https://node11.mainnet.pokt.network,https://node12.mainnet.pokt.network,https://node13.mainnet.pokt.network,https://node14.mainnet.pokt.network,https://node15.mainnet.pokt.network,https://node16.mainnet.pokt.network,https://node17.mainnet.pokt.network,https://node18.mainnet.pokt.network,https://node19.mainnet.pokt.network,https://node20.mainnet.pokt.network"
         WALLET_BASE_URL: "https://wallet.pokt.network"
         DASHBOARD_BASE_URL: "https://dashboard.pokt.network"
+        GATEWAY_BASE_URL: "https://gateway.pokt.network/v1/60676c9f7cbbfe002f0b9cbe"
```

Comment:
This should be

https://mainnet.gateway.pokt.network/...

And we should put origin protection on this endpoint for the wallet & explorer production URLs.

Reply:
Good catch, I will make sure I add some validation layer to check for this kind of invalid value. What do you mean by origin protection? Just this type of validation or?

Reply:
You can protect this endpoint from other people using it by setting an origin requirement.